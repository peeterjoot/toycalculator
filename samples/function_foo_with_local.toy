FUNCTION foo ( )
{
    INT16 v;
    v = 3;
    PRINT "In foo";
    PRINT v;
    RETURN;
};

PRINT "In main";
CALL foo();
PRINT "Back in main";
