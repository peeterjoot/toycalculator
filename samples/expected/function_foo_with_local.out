In main
In foo
3
Back in main
